<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cours</class>
 <widget class="QWidget" name="Cours">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/ADMIN/Cours.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>910</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="CoursTab">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>1260</width>
     <height>630</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QWidget" name="Ajout">
    <attribute name="title">
     <string>Ajouter Cours</string>
    </attribute>
    <widget class="QLabel" name="LNBRR_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>110</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de resevation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="IDC">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Cours</string>
     </property>
    </widget>
    <widget class="QLabel" name="LNomP">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom du cours</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NBRR">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>100</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LDateCours">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>230</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Cours</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DateCours">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>240</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NomC">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="HoraireC">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>280</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LHorairesC">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>270</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Horaire Cours</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>490</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonAjouter.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="Ajouter">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>490</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Afficher">
    <attribute name="title">
     <string>Afficher Cours</string>
    </attribute>
    <widget class="QTableWidget" name="AfficherCours">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>801</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>160</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>28</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID Cours</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom Cours</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Cours</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Horaire</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NBR Reservation</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="LNomP_2">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom Du Cours</string>
     </property>
    </widget>
    <widget class="QLabel" name="LHoraireC">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>240</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Horaire Cours</string>
     </property>
    </widget>
    <widget class="QLabel" name="LNBRR">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>310</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de resevation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IDP_2">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>80</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LIDP_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Cours</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NBR">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>300</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LDateC">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>200</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Cours</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="Horaire">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>250</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NomP_2">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>140</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="DateCours_2">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>210</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NBRR_3">
     <property name="geometry">
      <rect>
       <x>1320</x>
       <y>50</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>360</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonSupprimer.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>440</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonModifier.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="Enregistrer">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>440</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Supprimer">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>360</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Reservation</string>
    </attribute>
    <widget class="QTableWidget" name="AfficherR">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>761</width>
       <height>361</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>28</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID Personne</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID_Cours</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Reservation</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Abonne">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Abonnement">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Promotion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Stock">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
